### ISSUE TITLE
[Question] Why bundled dependencies appear in pom file (as runtime?)

### ISSUE THREAD

**Issue by kkocel** [GradleUp/shadow#694]
I wonder if it's necessary for dependencies to appear as `runtime` in the pom.xml file if they're already shadowed and bundled into the jar file. If it's not necessary then is there an option to exclude them from the pom file?

**Comment by dnault** [comment]
How are you configuring your publication? If you follow the [publishing example in the Users Guide](https://imperceptiblethoughts.com/shadow/publishing/) your pom.xml won't include the bundled dependencies.

**Comment by kkocel** [comment]
@dnault Yes, I did follow this example however, I had to manually skip runtime elements from the components:
```gradle
components.java.withVariantsFromConfiguration(configurations.runtimeElements) {
    skip()
}
```

**Comment by Goooler** [comment]
Try out newer versions.
